packages/dynamite/dynamite_end_to_end_test/test/tags_test.dart

```diff
@@ -0,0 +1,11 @@
+import 'package:dynamite_end_to_end_test/tags.openapi.dart';
+import 'package:test/expect.dart';
+import 'package:test/scaffolding.dart';
+
+void main() {
+  test('Nested clients', () {
+    final client = $Client(Uri());
+    expect(client.first, isA<$FirstClient>());
+    expect(client.second, isA<$SecondClient>());
+  });
```

Comment:
So we can be sure they really are identical.

```suggestion
  test('Nested clients', () {
    final client = $Client(Uri());
    expect(client.first, isA<$FirstClient>());
    expect(identical(client.first, client.first), isTrue);

    expect(client.second, isA<$SecondClient>());
    expect(identical(client.second, client.second), isTrue);
  });
```